mod-kb-ebsco/mod-kb-ebsco.postman_collection.json

```diff
@@ -18106,98 +18429,77 @@
+								"key": "X-Okapi-Token",
 								"value": "{{xokapitoken}}"
-							},
-							{
-								"key": "Content-Type",
-								"value": "application/vnd.api+json"
 							}
 						],
 						"body": {
 							"mode": "raw",
 							"raw": ""
 						},
 						"url": {
-							"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/packages/{{custom-packageid-for-resource}}",
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/configurations/entries/{{rm-api-url-id}",
 							"protocol": "{{protocol}}",
 							"host": [
 								"{{url}}"
 							],
 							"port": "{{okapiport}}",
 							"path": [
-								"eholdings",
-								"packages",
-								"{{custom-packageid-for-resource}}"
+								"configurations",
+								"entries",
+								"{{rm-api-url-id}"
 							]
 						},
-						"description": "Delete custom package which was set up for testing. This will also delete any custom titles that have been created as part of the custom package"
+						"description": "Clean-up RM-API Key configuration"
 					},
 					"response": []
-				}
-			],
-			"description": "Deletes newly created custom package which is used for resource testing. Deleting a custom package deletes all custom titles that are contained within the package ",
-			"event": [
-				{
-					"listen": "prerequest",
-					"script": {
-						"id": "e2aff0ec-6089-4e7d-b3bf-7803ac05390d",
-						"type": "text/javascript",
-						"exec": [
-							""
-						]
-					}
 				},
 				{
-					"listen": "test",
-					"script": {
-						"id": "80be542a-4821-49a5-b810-4b30cfd2d93a",
-						"type": "text/javascript",
-						"exec": [
-							""
-						]
-					}
-				}
-			]
-		},
-		{
-			"_postman_id": "c9160000-8719-4ccc-9a57-fc6ee2829ead",
-			"name": "tear-down titles test",
-			"item": [
-				{
-					"_postman_id": "04fdb7a8-4c90-4b64-ba51-47717fe20a93",
-					"name": "Delete Custom Package",
+					"name": "Check if configuration RM API customerid exists and should delete",
 					"event": [
 						{
 							"listen": "test",
 							"script": {
-								"id": "967dd204-5352-4475-940d-0d16f2b81eee",
+								"id": "7886c216-80c5-4093-8f4e-96d01b2cc90e",
 								"type": "text/javascript",
 								"exec": [
-									"pm.test(\"Status is 204\", function () {",
-									"    pm.response.to.have.status(204);",
-									"});"
+									"pm.test(\"Status is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Response must be valid and have a body\", function () {",
+									"    pm.response.to.be.ok; ",
+									"    pm.response.to.be.withBody;",
+									"    pm.response.to.be.json; ",
+									"});",
+									"",
+									"let jsonData = pm.response.json();",
+									"let customerIdExists = pm.environment.get(\"customerIdExists\");",
+									"if((jsonData !==null && jsonData.configs.length >0) && customerIdExists) {",
+									"    //custoemr id previously existed -- DO NOT DELETE (end execution)",
```

Comment:
a very minor nitpick - spelling of //customer

Reply:
Good eye - just corrected